taxi/forms.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "license_number", "first_name", "last_name", "email"
+        )
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if len(license_number) != 8:
+            raise forms.ValidationError(
+                "License number must be 8 characters long"
+            )
+        elif not (
+                license_number[:3].isupper() and license_number[:3].isalpha()
+        ):
+            raise forms.ValidationError(
+                "First 3 characters license number "
+                "must be are uppercase letters"

```

Comment:
The error message has a grammatical mistake. It should be: "First 3 characters of the license number must be uppercase letters."